File: Week 1 2 3/marsExploration/marsExploration.ts
const marsExploration2 = (s:string):number=> {
    let count:number = 0;
    let SOS:string = "SOS";
    for (let i = 0; i < s.length; i++) {
      if (s[i] !== SOS[i % 3]) {
        count++;
      }
    }
    return count;
  };
  
  const Array2 = "SOSSPSSQSSOR";
  console.log(marsExploration2(Array2));
  

File: Week 1 2 3/marsExploration/MarsExploration.java
package marsExploration;

public class MarsExploration {
    public static Number marsExploration(String s){
        Integer count = 0;
        String SOS= "SOS";
        for(int i = 0; i < s.length() ; i++ ){
            if(s.charAt(i) != SOS.charAt(i % 3)){
                count++;
            }
        }
        return count;
    }
}
